const anotherFunction = () =>{

    return (new Promise( (resolve, reject) => {
        if(false){
            resolve("resuelto");
        }else{
            reject("rechazado");
        }

    }));
}

anotherFunction()
    .then(response => console.log(response))
    .catch(err => console.log(err));